<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RabbitGIS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/Resource.qrc">
    <normaloff>:/icon/AppIcon</normaloff>:/icon/AppIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_track_T"/>
    <addaction name="actionExit_E"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options(&amp;O)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools_T">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <widget class="QMenu" name="menuMap_A">
     <property name="title">
      <string>Map(&amp;A)</string>
     </property>
     <property name="icon">
      <iconset resource="Resource/Resource.qrc">
       <normaloff>:/icon/Map</normaloff>:/icon/Map</iconset>
     </property>
    </widget>
    <addaction name="menuMap_A"/>
    <addaction name="actionMeasure_the_distance_M"/>
   </widget>
   <widget class="QMenu" name="menuView_V">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionStatusBar_S"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout_A"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuView_V"/>
   <addaction name="menuTools_T"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_O">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Map</normaloff>:/icon/Map</iconset>
   </property>
   <property name="text">
    <string>Open map(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>Open map</string>
   </property>
  </action>
  <action name="actionOpen_track_T">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Track</normaloff>:/icon/Track</iconset>
   </property>
   <property name="text">
    <string>Open track(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>Open track</string>
   </property>
  </action>
  <action name="actionExit_E">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Close</normaloff>:/icon/Close</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionMeasure_the_distance_M">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/Measure</normaloff>:/icon/Measure</iconset>
   </property>
   <property name="text">
    <string>Measure the distance(&amp;M)</string>
   </property>
   <property name="toolTip">
    <string>Measure the distance</string>
   </property>
  </action>
  <action name="actionStatusBar_S">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>StatusBar</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
